doc/user_guide/document_type_plugin_development_guide.md

```diff
@@ -70,13 +61,9 @@ The interface requires only a single method:
 protected abstract Stream<DocumentNode<DocumentVisitorType>> readDocuments(InputStreamWithResourceName loadedFile);
 ```
 
-Inside this method, you need to load the data and convert it into your class structure implementing the `DocumentNode` interfaces.
-The parameter `InputStreamWithResourceName` consists of an `InputStream` from which you can parse the data and a file name, that you can use for logging. 
+Inside this method, you need to load the data and convert it into your class structure implementing the `DocumentNode` interfaces. The parameter `InputStreamWithResourceName` consists of an `InputStream` from which you can parse the data and a file name, that you can use for logging.
 
-Implement the method so that it converts the `loadedFile` into a `Stream<DocumentNode<YOUR_TYPEDocumentNodeVisitor>>` by parsing it and using the factory for your class structure.
-If your datatype has only one document per file, simply return `Set.of(YOUR_DOCUMTNT_NODE)`.
-Otherwise it is important to stream the data and not to collect it in a List first. 
-For that implement an custom `Iterator` and `Iterable` and use:
+Implement the method so that it converts the `loadedFile` into a `Stream<DocumentNode<YOUR_TYPEDocumentNodeVisitor>>` by parsing it and using the factory for your class structure. If your datatype has only one document per file, simply return `Set.of(YOUR_DOCUMTNT_NODE)`. Otherwise it is important to stream the data and not to collect it in a List first. For that implement an custom `Iterator` and `Iterable` and use:
```

Comment:
```suggestion
Implement the method so that it converts the `loadedFile` into a `Stream<DocumentNode<YOUR_TYPEDocumentNodeVisitor>>` by parsing it and using the factory for your class structure. If your datatype has only one document per file, simply return `Set.of(YOUR_DOCUMTNT_NODE)`. Otherwise it is important to stream the data and not to collect it in a List first. For that implement a custom `Iterator` and `Iterable` and use:
```